src/Parser.php

```diff
@@ -7,17 +7,21 @@
 
 class Parser
 {
-    public static function getSchema()
+    public static function getSchema($exclude = [])
     {
-        return collect(DB::select('show tables'))->map(function ($row) {
-            $table = head($row);
+        return collect(DB::select('show tables'))
+            ->filter(function ($row) use ($exclude) {
+                return !in_array(head($row), $exclude);
+            })
+            ->map(function ($row) {
+                $table = head($row);
 
-            return [
-                'name' => $table,
-                'columns' => static::getColumns($table),
-                'indices' => static::getIndices($table),
-            ];
-        });
+                return [
```

Comment:
unexpected changes

Reply:
I try to change coding style to prevent this on my least commit a9e95d3